Remove unused config stuff

bfish713 opened the pull request:
Closes #<ISSUE_NUMBER>
<!-- See here for keywords in Github -->
<!-- https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword -->

### This PR: 
<!-- Describe what this PR adds to HotSHot -->
<!-- E.g. -->
<!-- * Implements feature 1 -->
<!-- * Implements feature 2 -->
<!-- * Fixes bug 3 -->

### This PR does not: 
<!-- Describe what is out of scope for this PR, if applicable.  Leave this section blank if it's not applicable -->
<!-- * Implement feature 3 because that feature is blocked by Issue 4   -->

### Key places to review: 
<!-- Describe key places for reviewers to pay close attention to -->
<!-- * file.rs, `add_integers` function -->

<!-- ### How to test this PR:  -->
<!-- Optional, uncomment the above line if this is relevant to your PR -->
<!-- If your PR can be tested through CI there is no need to add this section -->
<!-- * E.g. `just async_std test` -->

<!-- Complete the following items before creating this PR
* Are the proper people tagged to review it?
* Have you linked an issue to this PR?   -->

ss-es commented on crates/hotshot/src/traits/networking/libp2p_network.rs:
```diff
@@ -664,11 +636,6 @@ impl<K: SignatureKey + 'static> Libp2pNetwork<K> {
-                if is_da {
-                    handle.subscribe("DA".to_string()).await.unwrap();
-                }
-
```
Is the intention that everyone is always receiving all DA broadcasts/gossip?

ss-es reviewed (approved):
looks good to me, just one question

ss-es reviewed (changes requested):
oh, wait -- it looks like the PR broke `tests_3` somehow

feel free to dismiss

bfish713 commented:
Figured out test 3, basically the byzantine test for bad proposals is not great in general.  The network task effectively blocks us from sending the multiple bad proposals out.  The reason this PR broke it is because I removed round start delay which changed the timing such that the byzantine node blocks it's own proposal because it thinks it's a duplicate action.  I fixed this by adding more nodes so that we have enough honest leaders to complete 50 other views in time.  I feel like the byzantine leader timing out is totally a reasonable expectation but lmk if you think otherwise
